src/Workspaces/Core/Portable/ExternalAccess/Pythia/Api/PythiaAsyncLazyWrapper.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Pythia.Api
+{
+    internal struct PythiaAsyncLazyWrapper<T>
```

Comment:
❔ Why is this needed? Pythia would already have access to the `AsyncLazy<T>` defined in vs-threading, which is preferable for use in Visual Studio.

Reply:
I guess I can switch them to VS AsyncLazy. The only issue is that it takes JoinableTaskFactory optional parameter and if I don't pass it I get `VSTHRD012: Provide JoinableTaskFactory where allowed`.  I guess I can ignore that since there is the code is not doing anything with UI.